### PR TITLE
Updates SimpleAnalog sample

### PR DESCRIPTION
**Pull request by garanj**
Updates the basic `AnalogClock` example

- Include use of drop shadow
- Less bare-bones design
- Adds `UserConfigurations`

preview

### CODE REVIEW

File: WatchFaceFormat/SimpleAnalog/watchface/src/main/res/raw/watchface.xml [hunk 1]
```diff
@@ -22,29 +22,67 @@
...
+    </ColorConfiguration>
+  </UserConfigurations>
+  <Scene backgroundColor="#000000">
+    <Group x="100" y="100" width="250" height="250" name="background" alpha="127">
+      <Variant mode="AMBIENT" target="alpha" value="0" />
+      <PartDraw x="0" y="0" width="250" height="250">
+        <Ellipse x="0" y="0" width="250" height="250">
+          <Fill color="[CONFIGURATION.themeColor.3]" />
+        </Ellipse>
+        <Ellipse x="50" y="50" width="150" height="150">
```
Comment (yschimke) [inline WatchFaceFormat/SimpleAnalog/watchface/src/main/res/raw/watchface.xml:44]: Worth a comment?

<-- drop shadow -->

File: WatchFaceFormat/SimpleAnalog/watchface/src/main/res/raw/watchface.xml [hunk 2]
```diff
@@ -22,29 +22,67 @@
...
+      <!-- Shadows -->
+      <HourHand resource="hour" x="220" y="55" width="20" height="190" pivotX="0.5"
+        pivotY="0.92105263157" tintColor="#883c3c3c">
+        <Variant mode="AMBIENT" target="alpha" value="0" />
       </HourHand>
-      <!-- Ambient versions - no second hand in ambient mode -->
-      <MinuteHand resource="minute_hand_ambient" x="222" y="75" width="6" height="150" pivotX="0.5" pivotY="1" alpha="0">
-        <Variant mode="AMBIENT" target="alpha" value="255"/>
+      <MinuteHand resource="minute" x="222" y="30" width="16" height="220" pivotX="0.5" pivotY="0.9"
+        tintColor="#883c3c3c">
```
Comment (yschimke) [inline WatchFaceFormat/SimpleAnalog/watchface/src/main/res/raw/watchface.xml:59]: Should these move to theme also?

**Review by yschimke — APPROVED** [review]

File: WatchFaceFormat/SimpleAnalog/watchface/src/main/res/raw/watchface.xml (hunk 1 above)
Comment (garanj) [reply WatchFaceFormat/SimpleAnalog/watchface/src/main/res/raw/watchface.xml:45]: Done

File: WatchFaceFormat/SimpleAnalog/watchface/src/main/res/raw/watchface.xml (hunk 2 above)
Comment (garanj) [reply WatchFaceFormat/SimpleAnalog/watchface/src/main/res/raw/watchface.xml]: Done

**Comment by garanj** [comment]
Thanks Yuri!👍

**Pull request merged**